source/utilities.tex

```diff
@@ -27272,7 +27272,7 @@
 \remarks
 This constructor does not participate in overload resolution unless
 \begin{codeblock}
-declval<TimeZonePtr&>()->to_sys(local_time<Duration>{})
+decltype(declval<TimeZonePtr&>()->to_sys(local_time<Duration>{}))
```

Comment:
Do we really need / want a `decltype` here? "is (implicitly) convertible to" is defined for an expression and a destination type, not for a source type and a destination type (this actually matters in some cases, such as when the source expression is a bit-field and the destination is a reference type -- but probably not in this case).

I've opened #2029 for the general problem with the library using the meaningless term "is convertible to".

Reply:
I mostly wanted local consistency and to avoid suggesting that the expression is actually evaluated.